### PR TITLE
Standard Linter

### PR DESCRIPTION
**Pull request by marocchino**
riseshia/gannbaruzoi#35

### CODE REVIEW

File: build/vue-loader.conf.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ module.exports = {
       ? config.build.productionSourceMap
       : config.dev.cssSourceMap,
     extract: isProduction
-  }), { js: 'vue-ts-loader' }),
+  }), { js: ['../../loaders/standard-loader', 'vue-ts-loader'] }),
```
Comment (marocchino) [inline build/vue-loader.conf.js]: this relative path calc from actual vue filepath.
That's why I moved App.vue file.

File: loaders/standard-loader.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const linter = require('typescript-standard')
+
+module.exports = function standardLoader (text) {
+  this.cacheable()
```
Comment (riseshia) [inline loaders/standard-loader.js]: what's this?

**Review by riseshia — APPROVED** [review]

File: loaders/standard-loader.js (hunk 2 above)
Comment (marocchino) [reply loaders/standard-loader.js]: https://webpack.github.io/docs/how-to-write-a-loader.html#flag-itself-cacheable-if-possible

Comment (riseshia) [reply loaders/standard-loader.js]: OTL checked!

**Comment by riseshia** [comment]
(question) is this work?

```bash
(lint {1} ✓) gb-frontend yarn run test
yarn run v0.19.1
$ yarn run unit && yarn run e2e 
yarn run v0.19.1
$ cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run 
ts-loader: Using typescript@2.2.0 and /Users/user/Develop/gb-frontend/tsconfig.json
Hash: e0bdc8cc97632b01d813
Version: webpack 2.2.1
Time: 27ms

ERROR in /Users/user/Develop/gb-frontend/test/unit/specs/Hello.spec.ts
(2,19): error TS2307: Cannot find module 'src/components/Hello'.

ERROR in /Users/user/Develop/gb-frontend/src/main.ts
(5,17): error TS2307: Cannot find module './components/App'.
✨  Done in 4.22s.
yarn run v0.19.1
$ node test/e2e/runner.js 
ts-loader: Using typescript@2.2.0 and /Users/user/Develop/gb-frontend/tsconfig.json
Starting selenium server... started - PID:  42900

[Test] Test Suite
=====================

Running:  default e2e tests
> Listening at http://localhost:8080

 ✖ Timed out while waiting for element <#app> to be visible for 5000 milliseconds.  - expected "visible" but got: "not visible"
    at Object.defaultE2eTests [as default e2e tests] (/Users/user/Develop/gb-frontend/test/e2e/specs/test.js:13:8)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)


FAILED:  1 assertions failed (7.374s)

 _________________________________________________

 TEST FAILURE:  1 assertions failed, 0 passed. (7.546s)

 ✖ test

   - default e2e tests (7.374s)
   Timed out while waiting for element <#app> to be visible for 5000 milliseconds.  - expected "visible" but got: "not visible"
       at Object.defaultE2eTests [as default e2e tests] (/Users/shia/Develop/gb-frontend/test/e2e/specs/test.js:13:8)
       at _combinedTickCallback (internal/process/next_tick.js:67:7)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by marocchino** [comment]
hmm..
```
yarn test v0.19.1
$ yarn run unit && yarn run e2e
yarn run v0.19.1
$ cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run
ts-loader: Using typescript@2.2.0 and /Users/marocchino/Documents/gannbaruzoi-frontend/tsconfig.json
[]
[]
[]
[]
[]
16 02 2017 18:44:44.169:INFO [karma]: Karma v1.4.1 server started at http://0.0.0.0:9876/
16 02 2017 18:44:44.171:INFO [launcher]: Launching browser PhantomJS with unlimited concurrency
16 02 2017 18:44:44.184:INFO [launcher]: Starting browser PhantomJS
16 02 2017 18:44:45.308:INFO [PhantomJS 2.1.1 (Mac OS X 0.0.0)]: Connected on socket tr2zS-XFsCSdNYDlAAAA with id 79168398
PhantomJS 2.1.1 (Mac OS X 0.0.0) INFO LOG: 'You are running Vue in development mode.
Make sure to turn on production mode when deploying for production.
See more tips at https://vuejs.org/guide/deployment.html'
ERROR LOG: '[Vue warn]: Failed to mount component: template or render function not defined.
(found in root instance)'

  Hello.vue
    ✗ should render correct contents
	undefined is not a constructor (evaluating 'vm.$el.querySelector('.hello h1')')
	index.ts:13828:43


PhantomJS 2.1.1 (Mac OS X 0.0.0): Executed 1 of 1 (1 FAILED) ERROR (0.011 secs / 0.004 secs)


=============================== Coverage summary ===============================
Statements   : 100% ( 0/0 )
Branches     : 100% ( 0/0 )
Functions    : 100% ( 0/0 )
Lines        : 100% ( 0/0 )
================================================================================
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
```

```
$ yarn run e2e
yarn run v0.19.1
$ node test/e2e/runner.js
ts-loader: Using typescript@2.2.0 and /Users/marocchino/Documents/gannbaruzoi-frontend/tsconfig.json
Starting selenium server... []
started - PID:  1847

[Test] Test Suite
=====================

Running:  default e2e tests
[]
[]
[]
> Listening at http://localhost:8080

 ✔ Element <#app> was visible after 80 milliseconds.
 ✔ Testing if element <.hello> is present.
 ✔ Testing if element <h1> contains text: "Welcome to Your Vue.js App".
 ✔ Testing if element <img> has count: 1

OK. 4 assertions passed. (8.251s)

✨  Done in 12.22s.
```

**Review by riseshia — APPROVED** [review]

**Pull request merged**